lumicks/pylake/calibration.py

```diff
@@ -310,12 +360,57 @@ def from_field(hdf5, force_channel, time_field="Stop time (ns)") -> "ForceCalibr
         items = []
         for calibration_item in hdf5["Calibration"].values():
             if force_channel in calibration_item:
-                attrs = calibration_item[force_channel].attrs
+                attrs = dict(calibration_item[force_channel].attrs)
                 if time_field in attrs.keys():
+                    # Copy the timestamp at which the calibration was applied into the item
+                    attrs["Timestamp (ns)"] = calibration_item.attrs.get("Timestamp (ns)")
                     items.append(ForceCalibrationItem(attrs))
 
         return ForceCalibration(time_field=time_field, items=items)
 
+    def print_summary(self, tablefmt):
+        return tabulate(
+            (
+                (
+                    idx,
+                    datetime.datetime.fromtimestamp(int(item.applied_at / 1e9)).strftime("%x %X")
+                    if item.applied_at
+                    else "-",
```

Comment:
nitpick: curious, if you wrap this in `()` does it stay on one line? would be nice, but if not it's fine like this

Reply:
```
                    (
                        datetime.datetime.fromtimestamp(int(item.applied_at / 1e9)).strftime(
                            "%x %X"
                        )
                        if item.applied_at
                        else "-",
                        item.kind,
                    ),
```
😬 

Actually, let me put that thing in a tiny inline function. If I just define:
```python
        def format_timestamp(timestamp):
            return datetime.datetime.fromtimestamp(int(timestamp)).strftime("%x %X")
```

right before, then that blurb becomes far more readable.